src/common/config.h

```diff
@@ -9,13 +10,29 @@ namespace raftcpp {
 
 class Config final {
     public:
-    static Config From(const std::string &config_str) { return Config(); }
+    static Config From(const std::string &config_str) {
+        static Config config;
```

Comment:
I think this `config` shouldn't be local static.

Reply:
Done